js/views/startScreen.js

```diff
@@ -25,6 +25,10 @@ var StartScreen = React.createClass({
     this.handleResize();
   },
 
+  componentWillReceiveProps: function() {
```

Comment:
@mistermendez By doing below also can fix the problem. But we will be duplicating the code.

  componentWillReceiveProps: function(nextProps) {
    if (nextProps.contentTree.description == this.props.contentTree.description) {
      if (ReactDOM.findDOMNode(this.refs.description)){
        this.setState({
          descriptionText: Utils.truncateTextToWidth(ReactDOM.findDOMNode(this.refs.description), nextProps.contentTree.description)
        });
      }
    }
  },